src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/RegionAnalysisTests.vb

```diff
@@ -5963,6 +6011,8 @@ End Structure
             Assert.Equal(Nothing, GetSymbolNamesJoined(dataFlowAnalysisResults.VariablesDeclared))
             Assert.Equal(Nothing, GetSymbolNamesJoined(dataFlowAnalysisResults.AlwaysAssigned))
             Assert.Equal("a, b", GetSymbolNamesJoined(dataFlowAnalysisResults.Captured))
+            Assert.Equal("a, b", GetSymbolNamesJoined(dataFlowAnalysisResults.CapturedInside))
+            Assert.Equal("a, b", GetSymbolNamesJoined(dataFlowAnalysisResults.CapturedOutside))
```

Comment:
Why do we get "a, b" for as CapturedInside and CapturedOutside?
I think bug in ReadWriteWalker.vb line 43

Reply:
ReadWriteWalker.vb line 43 just assign an empty list if the region is erroneous. 

I think it's because the variables are literally captured inside *and* outside, just like WrittenInside and WrittenOutside in the same test.